lib/versioning/ubuntu/index.ts

```diff
@@ -0,0 +1,149 @@
+import { NewValueConfig, VersioningApi } from '../common';
+
+export const id = 'ubuntu';
+export const displayName = 'Ubuntu';
+export const urls = ['https://changelogs.ubuntu.com/meta-release'];
+export const supportsRanges = false;
+
+// validation
+
+function isValid(input: string): string | boolean | null {
+  return (
+    typeof input === 'string' &&
+    /^(0[4-5]|[6-9]|[1-9][0-9])\.[0-9][0-9](\.[0-9]{1,2})?$/.test(input)
+  );
+}
+
+function isVersion(input: string): string | boolean | null {
+  return isValid(input);
+}
+
+function isCompatible(
+  version: string,
+  _range?: string
+): string | boolean | null {
+  return isValid(version);
```

Comment:
Not sure what option is the best here

Reply:
Most version maps `isCompatible` to `isVersion`, which is what you're doing transitively